pkg/component/gardeneradmissioncontroller/admissioncontroller.go

```diff
@@ -44,6 +49,15 @@ var TimeoutWaitForManagedResource = 5 * time.Minute
 
 // Values contains configuration values for the gardener-admission-controller resources.
 type Values struct {
+	ClientConnection               ClientConnection
+	LogLevel                       string
```

Comment:
For GAPI, we also have `LogFormat`.

Reply:
Can you tell why we expose the`LogFormat`? We don't plan to expose this setting via API (yet), so there is probably no point in making it configurable?